assets/schema_genomeinfo.json

```diff
@@ -16,8 +16,8 @@
             "genome_fna": {
                 "type": "string",
                 "format": "file-path",
-                "pattern": "^\\S+\\.gz$",
-                "errorMessage": "Fasta file must be provided, the name cannot contain spaces and must have extension '.gz'",
```

Comment:
We got comments for metatdenovo manuscript about not allowing unzipped files for the samples. To me, it is reasonable to now have unzipped genomes as they might be many and really heavy in terms of storage. but I wanted just to say since it happened to be commented on.

Reply:
Funny, I thought I changed this...

Reply:
Now I see, I updated the fasta file but not the gff. Upcoming in the next push (waiting for tests to fail).